keras/layers/reshaping/repeat_vector_test.py

```diff
@@ -29,3 +29,19 @@ def test_repeat_vector_with_dynamic_dimension(self):
         input_layer = layers.Input(batch_shape=(2, None))
         repeated = layers.RepeatVector(n=3)(input_layer)
         self.assertEqual(repeated.shape, (2, 3, None))
+
+    def test_repeat_vector_with_invalid_n(self):
```

Comment:
For checking errors like this where there are only two possible cases (int or non-int), it is enough to check a single non-int value.

When there are more cases (type A, type B, other) you'd check one value per type, then an "other" type.